cms/static/js/views/active_video_upload_list.js

```diff
@@ -212,13 +231,16 @@ define([
             },
 
             showErrorMessage: function(errorMsg) {
-                var titleMsg = gettext('Your file could not be uploaded');
-                this.fileErrorMsg = new NotificationView.Error({
-                    title: titleMsg,
-                    message: errorMsg
-                });
-                this.fileErrorMsg.show();
-                this.readMessages([titleMsg, errorMsg]);
+                var titleMsg;
+                if (!this.fileErrorMsg) {
+                    titleMsg = gettext('Your file could not be uploaded');
+                    this.fileErrorMsg = new NotificationView.Error({
+                        title: titleMsg,
+                        message: errorMsg
+                    });
+                    this.fileErrorMsg.show();
+                    this.readMessages([titleMsg, errorMsg]);
```

Comment:
Why not? I mean, if we have uploaded 2 files with non-ascii chars, it will only show a single error message? Why can't we remove the existing one and show the message? But I think its a separate discussion. I am Ok with it if this is a requirement. 

secondly, `if (!this.fileErrorMsg)` is not informative and readable, Moving this to a method like `if (!ErrorMessageShownAlready())` will make it more readable.